web/src/app/util/avatar/BaseAvatar.js

```diff
@@ -57,11 +57,11 @@ export default class BaseAvatar extends Component {
     const { userID, fallback, ...props } = this.props
 
     if (this.state.valid) {
-      return <Avatar src={this._image.src} {...props} />
+      return <Avatar alt='' src={this._image.src} {...props} />
```

Comment:
Why the empty alt string, isn't that the default?

Reply:
It means that the image is decorative so the screen reader should ignore it.